### ISSUE TITLE
Alert List text appears to be a heading

### ISSUE THREAD

**Issue by martypowell** [baltimorecounty/BCPL-assets#533]
https://www.bcpl.info/alerts.html

I think this may be the intended behavior, but the text is awfully aggressive. 

fyi @danfox01 @sdurphy

**Comment by jdomasky** [comment]
@martypowell I agree that the alert text on the Alerts page is too aggressive, especially because most alerts will be holiday closings rather than emergencies.
@danfox01 @sdurphy It might be more user-friendly if we reduce the text size to 18px, similar to the database titles on the Research Databases page:
https://www.bcpl.info/books-and-more/databases.html

**Comment by sdurphy** [comment]
@jdomasky I definitely thin k 18px would work. @martypowell can we make that change without effecting cards in other sections?
![screen shot 2018-06-27 at 10 44 14 am](https://user-images.githubusercontent.com/2189638/41981517-1b007604-79f7-11e8-8523-dde7b41f0ddc.png)

**Comment by martypowell** [comment]
@sdurphy I've added a class to the list of the alerts, so that we can target this specific list.

Personally even at 18 px, the bold still seems agressive. It also seems to me that this page may benefit from a brief one or two sentence overview

**Comment by jdomasky** [comment]
@sdurphy, an alert was activated today [in prod](https://www.bcpl.info/alerts.html) that will be there for a week. I'll leave it up to @danfox01 to decide if this is an issue another developer can (should?) pick up.

**Comment by danfox01** [comment]
Not urgent. Sam, if you want to keep working on this with @jsmith176 that is fine, but I think this can easily wait, too.

**Comment by jsmith176** [comment]
Keeping with the theme here. How about a white icon with bold dark teal text?

![image](https://user-images.githubusercontent.com/2495453/43077858-a7a029f6-8e56-11e8-9809-4c4b6bac2478.png)

**Comment by martypowell** [comment]
Where are we at with this?

**Comment by danfox01** [comment]
Just been chillin since you've been out. I'm fine with whatever styling you and @sdurphy decide on. I will also add a little bit of intro copy like you suggested.

**Comment by martypowell** [comment]
@sdurphy this is on you, we can make the changes once you provide them.
